controller/userTypeController.js

```diff
@@ -22,7 +26,28 @@ const getAll = (req, res) => {
     });
 };
 
+const retrieveById = (req, res) => {
+    const userTypeId = req.body.id;
```

Comment:
- `retrieveById` should be a GET request. As such the ID is sent as a URL parameter, not in the body.
- User check is missing.

Reply:
Outdated commit